RecoTracker/LSTCore/interface/LST.h

```diff
@@ -1,8 +1,8 @@
-#ifndef RecoTracker_LSTCore_interface_alpaka_LST_h
-#define RecoTracker_LSTCore_interface_alpaka_LST_h
+#ifndef RecoTracker_LSTCore_interface_LST_h
```

Comment:
this file is used only in alpaka level (src or plugins); does it actually need to be moved?

Reply:
Oh that's true. But maybe it should still stay out since there's no `ALPAKA_ACCELERATOR_NAMESPACE` dependence. (see https://github.com/cms-sw/cmssw/pull/45117#discussion_r1674643352)